docs/faq/network/network-configuration.md

```diff
@@ -21,6 +21,7 @@ The following ports will need to be opened to run the blockchain node correctly:
 | ERIGON     | Execution Chain Data Channel              | 30303 | X   |     |
 | ERIGON     | Execution Chain Discovery                 | 30303 |     | X   |
 | LIGHTHOUSE | Beacon Communication and Data             | 9000  | X   | X   |
+| TEKU       | Beacon Communication and Data             | 9000  | X   | X   |
```

Comment:
I think it would be better to just put the link to each clients config page so we don't have to maintain these numbers. It is not super likely that they change ports but I don't think it is our responsibility to maintain this list, client page also have more context.

I suggest to:

1. Delete this table above
2. Keep only the link below - they also provide more information and context
3. Update the geth link to: https://geth.ethereum.org/docs/fundamentals/security
4. The Lighthouse link seems to not work for the anchor / highlight, I suggest to replace it with: https://lighthouse-book.sigmaprime.io/advanced_networking.html#how-to-open-ports